package.json

```diff
@@ -30,8 +30,8 @@
         "startWebpackServer": "webpack-dev-server --no-live-reload --config ./itsJustJavascript/webpack.config.js",
         "startSiteFront": "webpack-dev-server --config ./itsJustJavascript/webpack.config.js",
         "startLocalBakeServer": "http-server ./localBake -p 3000",
-        "testBundlewatch": "ENV=production webpack --config ./itsJustJavascript/webpack.config.js -p && bundlewatch --config .bundlewatch.config.json",
-        "testBundlewatchJson": "ENV=production webpack --config ./itsJustJavascript/webpack.config.js -p --json > itsJustJavascript/webpack/webpack-bundle-stats.json && bundlewatch --config .bundlewatch.config.json",
+        "testBundlewatch": "ENV=production webpack --config ./itsJustJavascript/webpack.config.js -p && cat itsJustJavascript/webpack/commons\\~*.js > itsJustJavascript/webpack/commons.js && bundlewatch --config .bundlewatch.config.json",
+        "testBundlewatchJson": "ENV=production webpack --config ./itsJustJavascript/webpack.config.js -p --json > itsJustJavascript/webpack/webpack-bundle-stats.json && cat itsJustJavascript/webpack/commons\\~*.js > itsJustJavascript/webpack/commons.js && bundlewatch --config .bundlewatch.config.json",
```

Comment:
Note that in order to get `bundlewatch` to work again, all chunks get concatenated into a single `commons.js` which may potentially not be valid JS, but is useful to estimate total file sizes.

Potentially, we could rename this to something like `commons--merged--bundlewatch-only.js` so that it's more obvious that it isn't supposed to be used.

Reply:
Leaving this for later 👍